articles/templating-your-static-site/index.md

```diff
@@ -0,0 +1,188 @@
+
+**Tip:** Another naming convention that you may want to use is to add -layout at the end of your layout files. This helps to differentiate them from your partials.
+
+In the `_includes` folder, create a file called `_base-layout.ejs` file and add the following to it:
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+    <% include _head.ejs %>
+</head>
+<body>
+    <% include _header.ejs %>
+    <main>
+        <%- content %>
+    </main>
+    <% include _footer.ejs %>
+</body>
+</html>
+```
+If you're new to EJS, `<% include` is the code (syntax) for including a partial file and `<%-` is for including an EJS variable. Content is a variable Eleventy uses to add the body content of HTML, Markdown and other input files.
+
+Congratulations, you've just created your first Eleventy layout file.
+
+###  Rendering Your First Layout File
+
+We have our basic layout template but we need to tell Eleventy where to use it. This is where front-matter comes in.
+
+Front-Matter allows you to define variables you can use in your layout templates both built-in (from Eleventy) and those you've created yourself. By default, Eleventy uses YAML for front-matter though you can change it to JSON or even JavaScript if you prefer.
+
+The most basic front-matter variable, layout will instruct Eleventy which layout file to use to render a page.
+
+On one of the HTML pages of your site, remove the code you included in EJS partials (head, header and footer if you followed the tutorial.)
+
+Then, add the following layout front-matter at the beginning:
+
+```yaml
+---
+layout: _base-layout.ejs
+---
+```
+The three dashes define the start and the end of the front-matter.
+
+Run `npx @11ty/eleventy --serve` in the terminal to start a local web-server for your site that will reload on changes.
+
+Go to `http://localhost:8080/nameofhtmlfile` to view the page. It should display the head, header and footer as well as the content of the page.
+
+Congratulations, you've successfully converted your first static HTML page to use an EJS Eleventy layout file.
+
+## Adding More Front Matter
+
+Remember how we templated your website's header so that all the meta tags would be the same? Now that you've added your first piece of front-matter, we can add some more to account for unique values like different page titles.
+
+First, In your partial pages (such as `_head.ejs`), remove everything inside the content attribute of your meta tags and replace it with EJS variables like the example below:
+
+```html
+<meta property="og:title" content="<%- title %>">
+<meta property="og:description" content="<%- short_description %>">
+```
+Adding - after <% definites it as an EJS variable rather than a partial file which we did before. The values such as title and short-description are the names of the variables which we will use in the front-matter.
+
+Next, in one of your HTML pages, add the EJS variables you've just created and the content you want to add like so:
```

Comment:
Either "in the HTML for one of your pages" or "on one of your HTML pages" would be more correct.

Reply:
Chose on one of your HTML pages.